src/js/components/tasks/capture/changes.tsx

```diff
@@ -130,83 +87,45 @@ const CaptureModal = ({ orgId, changeset, isOpen, toggleModal }: Props) => {
     }
   };
 
-  const handleSubmitClicked = () => {
-    // Click hidden button inside form to activate native browser validation
-    /* istanbul ignore else */
-    if (submitButton.current) {
-      submitButton.current.click();
-    }
-  };
-
-  const handleClose = () => {
-    toggleModal(false);
-    resetForm();
-  };
-
-  const submitChanges = (e: React.FormEvent<HTMLFormElement>) => {
-    setCapturingChanges(true);
-    handleSubmit(e);
-  };
-
   const totalChanges = Object.values(changeset).flat().length;
-  const changesChecked = Object.values((inputs as Inputs).changes).flat()
-    .length;
+  const changesChecked = Object.values(inputs.changes).flat().length;
   const allChangesChecked = changesChecked === totalChanges;
   const noChangesChecked = !changesChecked;
 
   return (
-    <Modal
-      isOpen={isOpen}
-      size="medium"
-      disableClose={capturingChanges}
-      heading={i18n.t('Select the changes to capture')}
-      footer={[
-        <Button
-          key="cancel"
-          label={i18n.t('Cancel')}
-          disabled={capturingChanges}
-          onClick={handleClose}
-        />,
-        <Button
-          key="submit"
-          type="submit"
-          label={
-            capturingChanges ? (
-              <LabelWithSpinner
-                label={i18n.t('Capturing Selected Changes…')}
-                variant="inverse"
-              />
-            ) : (
-              i18n.t('Capture Selected Changes')
-            )
-          }
-          variant="brand"
-          onClick={handleSubmitClicked}
-          disabled={!changesChecked || capturingChanges}
-        />,
-      ]}
-      onRequestClose={handleClose}
+    <form
+      className="slds-form slds-p-around_large has-checkboxes"
+      data-form="task-capture"
     >
-      <form
-        className="slds-form"
-        data-form="task-capture"
-        onSubmit={submitChanges}
-      >
-        <div className="slds-scrollable_y slds-p-around_large">
-          <div className="form-grid slds-p-around_x-small">
+      <ModalCard>
+        <pre>
```

Comment:
Surprised that `pre` doesn't create line-break issues. If this is mainly for monospace type, I'd probably just replace the span with a `code` tag?